api/.jshintrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Shouldn't file be named .eslintrc?

Reply:
Or .eslintrc.json. I'm only sure that file shouldn't be named .jshintrc, as we don't use it.

Reply:
My mistake. I was confused.

Reply:
Glad you figured it out. The .jshint  file is complementing the linting errors.